src/guides/v2.4/graphql/queries/categories.md

```diff
@@ -0,0 +1,205 @@
+
+**Request:**
+
+```graphql
+{
+  categories(filters: {ids: {in: ["3", "9", "11", "20", "37", "38"]}} pageSize:3 currentPage: 2){
+    total_count
+    items {
+      id
+      level
+      name
+      path
+      children_count
+      children {
+        id
+        level
+        name
+        path
+        children_count
+        children {
+          id
+          level
+          name
+          path
+        }
+      }
+    }
+    page_info {
+      current_page
+      page_size
+      total_pages
+    }
+  }
+}
+```
+
+**Response:**
+
+```json
+{
+  "data": {
+    "categories": {
+      "total_count": 6,
+      "items": [
+        {
+          "id": 20,
+          "level": 2,
+          "name": "Women",
+          "path": "1/2/20",
+          "children_count": "8",
+          "children": [
+            {
+              "id": 22,
+              "level": 3,
+              "name": "Bottoms",
+              "path": "1/2/20/22",
+              "children_count": "2",
+              "children": [
+                {
+                  "id": 27,
+                  "level": 4,
+                  "name": "Pants",
+                  "path": "1/2/20/22/27"
+                },
+                {
+                  "id": 28,
+                  "level": 4,
+                  "name": "Shorts",
+                  "path": "1/2/20/22/28"
+                }
+              ]
+            },
+            {
+              "id": 21,
+              "level": 3,
+              "name": "Tops",
+              "path": "1/2/20/21",
+              "children_count": "4",
+              "children": [
+                {
+                  "id": 23,
+                  "level": 4,
+                  "name": "Jackets",
+                  "path": "1/2/20/21/23"
+                },
+                {
+                  "id": 24,
+                  "level": 4,
+                  "name": "Hoodies & Sweatshirts",
+                  "path": "1/2/20/21/24"
+                },
+                {
+                  "id": 25,
+                  "level": 4,
+                  "name": "Tees",
+                  "path": "1/2/20/21/25"
+                },
+                {
+                  "id": 26,
+                  "level": 4,
+                  "name": "Bras & Tanks",
+                  "path": "1/2/20/21/26"
+                }
+              ]
+            }
+          ]
+        },
+        {
+          "id": 37,
+          "level": 2,
+          "name": "Sale",
+          "path": "1/2/37",
+          "children_count": "0",
+          "children": []
+        },
+        {
+          "id": 38,
+          "level": 2,
+          "name": "What's New",
+          "path": "1/2/38",
+          "children_count": "0",
+          "children": []
+        }
+      ],
+      "page_info": {
+        "current_page": 2,
+        "page_size": 3,
+        "total_pages": 2
+      }
+    }
+  }
+}
+```
+
+## Input attributes
+
+You must specify the `filters` attribute as input to your query.
```

Comment:
filters is not required. if it is omitted the store's default root category is returned (same as for categoryList)